constants/areaCodes.py

```diff
@@ -0,0 +1,422 @@
+AREA_CODES = [
```

Comment:
can we have a list of area codes for each location, similar to how everything else is separated by city/posting ? This way the area code would match the posting

Reply:
Definitely possible, but with most people's phones being cell phones, area codes matter a lot less. I'd be wary to limit out phone numbers to just 6 area codes (I checked :p)